libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/map/MapBinder.kt

```diff
@@ -44,10 +44,19 @@ internal class MapBinder(
     private val mapView: MapView
 ) : UIBinder {
 
-    init {
-        mapView.compass.enabled = false
-        mapView.scalebar.enabled = false
-    }
+    private val compassComponent = CompassComponent(
+        context.context,
+        mapView,
+        context.styles.mapCompassParams,
+        context.maneuverBehavior.maneuverViewHeight,
+        context.systemBarsInsets
+    )
+    private val scalebarComponent = ScalebarComponent(
+        mapView,
+        context.styles.mapScalebarParams,
+        context.maneuverBehavior.maneuverViewHeight,
```

Comment:
@dzinad I haven't yet run the latest changes, but based on the code it looks like we are moving the `Scalebar` below `ManeuverView` as well when it becomes visible. However, we should not do that with the `Scalebar`. It will always stick to the top and be hidden beneath the `ManeuverView` in case we start active navigation and `ManeuverView` becomes visible.

Reply:
> It will always stick to the top and be hidden beneath the ManeuverView in case we start active navigation and ManeuverView becomes visible.

Really? Why?